cypress/integration/plugins/security-dashboards-plugin/change_tenant_successfully.js

```diff
@@ -12,6 +12,7 @@ if (Cypress.env('SECURITY_ENABLED')) {
 
     before(() => {
       cy.server();
+      localStorage.setItem('home:welcome:show', false);
```

Comment:
```suggestion
      localStorage.setItem('home:welcome:show', false);
      localStorage.setItem('home:newThemeModal:show', false);
```

Reply:
i think technically not needed anymore for 2.14 since we reverted but won't hurt.